src/components/Date/SimpleDate.jsx

```diff
@@ -0,0 +1,211 @@
+
+  // If the active element is _not_ part of the SimpleDate, call onBlur
+  const blurIfOutsideField = () => {
+    // Push the function onto the event loop because otherwise,
+    // document.activeElement will be <body>, not the _actual_ active element.
+    setTimeout(() => {
+      if (
+        ![
+          `#${inputId.current} [name="${name}Month"]`,
+          `#${inputId.current} [name="${name}Day"]`,
+          `#${inputId.current} [name="${name}Year"]`,
+        ].some(
+          selector =>
+            document.querySelector(selector) === document.activeElement,
+        )
+      )
+        onBlur();
+    });
+  };
+
+  return (
+    <fieldset className={fieldsetClass} id={inputId.current}>
+      <legend className="vads-u-font-size--base vads-u-font-weight--normal">
+        {label || 'Date of birth'}
+        {required && <span className="form-required-span">(*Required)</span>}
+      </legend>
+      {errorSpan}
+      <div
+        className={
+          !errorMessage ? undefined : 'usa-input-error form-error-date'
+        }
+      >
+        <div className="usa-date-of-birth usa-datefields clearfix">
+          <div className="form-datefield-month">
+            <Select
+              errorMessage={errorMessage && ''}
+              label="Month"
+              name={`${name}Month`}
+              options={months}
+              value={month}
+              onValueChange={update => {
+                handleChange('month', update, date, onValueChange);
+              }}
+              ariaDescribedby={ariaDescribedbyWithError}
+              onBlur={blurIfOutsideField}
+            />
+          </div>
+          <div className="form-datefield-day">
+            <Select
+              errorMessage={errorMessage && ''}
+              label="Day"
+              name={`${name}Day`}
+              options={daysForSelectedMonth}
+              value={day}
+              onValueChange={update => {
+                handleChange('day', update, date, onValueChange);
+              }}
+              ariaDescribedby={ariaDescribedbyWithError}
+              onBlur={blurIfOutsideField}
+            />
+          </div>
+          <div className="usa-datefield usa-form-group usa-form-group-year">
+            <NumberInput
+              errorMessage={errorMessage && ''}
+              label="Year"
+              name={`${name}Year`}
+              max={maxYear}
+              min={minYear}
+              pattern="[0-9]{4}"
+              field={year}
+              onValueChange={update => {
+                handleChange('year', update, date, onValueChange);
+              }}
+              ariaDescribedby={ariaDescribedbyWithError}
+              onBlur={blurIfOutsideField}
+            />
+          </div>
+        </div>
+      </div>
+    </fieldset>
+  );
+};
+
+const dateFragment = PropTypes.shape({
+  value: PropTypes.string.isRequired,
+  dirty: PropTypes.bool.isRequired,
```

Comment:
This is never even used, but it's there in the old version, so :man_shrugging: 

Subject to refactoring later.

Reply:
`:old-man-yells-at-cloud:`